dockers/Screenshotter/screenshotter.js

```diff
@@ -379,22 +424,28 @@ function takeScreenshot(key) {
         return;
     }
 
-    let file = path.join(dstDir, key + "-" + opts.browser + ".png");
+    let basename = key + "-" + opts.browser;
+    if (itm.quirky && itm.quirky.indexOf(mode) !== -1) {
+        // a test case known to differ depending on mode may make use of this
+        basename += "-" + qabbr[mode];
+    }
+    let file = path.join(dstDir, basename + ".png");
     let retry = 0;
     let loadExpected = null;
     if (opts.verify) {
         loadExpected = promisify(fs.readFile, file);
     }
 
-    const url = katexURL + "test/screenshotter/test.html?" + itm.query;
+    const url = katexURL + "ss-render.html?mode=" + mode + "&" + itm.query;
     driver.call(loadMath);
 
     function loadMath() {
         if (!opts.reload && driverReady) {
             driver.executeAsyncScript(
                     "var callback = arguments[arguments.length - 1]; " +
                     "handle_search_string(" +
-                    JSON.stringify("?" + itm.query) + ", callback);")
+                    JSON.stringify(
+                        "?mode" + mode + "&" + itm.query) + ", callback);")
```

Comment:
After reading through the code in `ss-render.html` I see that it doesn't make use of `mode` and actually the only thing that does anything with `mode` is `takeScreenshots` which determines which doctype string to prepend the `ss-render.html` with.  It probably doesn't need to be included in the `url` either by I'm going to leave it there as a cache busting mechanism.